docs/tools.md

```diff
@@ -553,6 +553,27 @@ It prints the MKey
 ```
 
 
+## mt-parrot
+
+```
+generate deterministic metrics for each metrictank partition
+
+Usage:
+  parrot [flags]
+
+Flags:
+      --artificial-metrics-interval duration   interval to send metrics (default 5s)
+      --gateway-address string                 the url of the metrics gateway (default "http://localhost:6059")
+      --gateway-key string                     the bearer token to include with gateway requests
+  -h, --help                                   help for parrot
+      --log-level string                       log level. panic|fatal|error|warning|info|debug (default "info")
+      --org-id int                             org id to publish parrot metrics to (default 1)
+      --partition-count int32                  number of partitions to publish parrot metrics to (default 8)
+      --partition-method string                the partition method to use, must be one of bySeries|bySeriesWithTags|bySeriesWithTagsFnv (default "bySeries")
```

Comment:
"to use" is confusing here. more like "to mimic". or "partition method in use on the gateway"
ditto for partition-count. should be partitioning enabled on the gateway.

Reply:
partition count is determined by kafka, right?

I don't see a mt-gateway flag, was pretty sure it's determined by the number of kafka partitions

Reply:
> partition count is determined by kafka, right?

yes, but really by the admin of the platform.
so maybe call it "kafka partitions in use"
if we call it "number of partitions to publish parrot metrics to " then it implies that you control here how many partitions we publish to, which is not true, we don't have the power to control that

Reply:
30ca761bb532cfd4e99ecb831eb6ca632a6ba0e2